docs/data-warehouse-api/resources/tracker/raw_data.md

```diff
@@ -71,90 +84,127 @@ curl -X 'POST' \
+| to                 | date/time     | The end date/time, indicating up to which moment messages should be retrieved into the file. This relates to the message time—when the packet was registered by a tracker. The specified date must be after the "from" date. The time is specified along with the time zone according to ISO 8601.                     | "2023-08-24T08:04:36Z"                                                                                                                                                            | yes      |
+| columns            | string array  | List of attributes to retrieve in CSV/Parquet file. You can obtain it with [get_inputs](./raw_data.md#get_inputs) request.                                                                                                                                                                                             | `["flags.location_valid","lat","lng","discrete_inputs.1","inputs.board_voltage"]`                                                                                                 | yes      |
+| server_time_filter | string/object | An interval for additional filtering of messages by server time. If used, messages will be returned not only based on the message time (when the packet was registered by a tracker), but also filtered by the server time (when the message was received by the server).                                              | `"2024-02-03T10:26:26+0500/2024-02-03T10:27:18+0500"` / `{"from": "2024-02-03T10:26:26+0500", "to": "2024-02-03T10:27:18+0500"}` / `{"interval":"2024-02-03T10:26:26+0500/PT1H"}` | no       |
+| format             | string        | This is used when you need to request data in `parquet` format. If the format parameter is not provided, the default format used will be CSV.                                                                                                                                                                          | `"parquet"`                                                                                                                                                                       | no       |
 
-!!! note "Instead of using `from`/`to` parameters it is possible to set `interval` parameter - ISO 8601 formatted interval, for example 2023-08-24T08:04:36.306Z/PT24H."
+!!! note "Instead of using the `from`/`to` parameters, you can set the `interval` parameter — an ISO 8601 formatted interval, for example, `2023-08-24T08:04:36.306Z/PT24H`."
+
+The response could be provided in a CSV or [Parquet](https://parquet.apache.org/docs/overview/) format file, with columns 
+predefined in the `columns` parameter of the API request. 
 
-The response is provided in a CSV format file, with columns that are predefined in the `columns` parameter of the API request.
 Here are the specifications for the table output:
```

Comment:
@Astarsys these specifications related only to CSV format

Reply:
Yes, thank you. I will add this to my next commit